Formula/llvm.rb

```diff
@@ -130,6 +130,7 @@ class Llvm < Formula
   option "with-python", "Build bindings against custom Python"
   option "with-shared-libs", "Build shared instead of static libraries"
   option "without-libffi", "Do not use libffi to call external functions"
+  option "without-experimental", "Do not build libc++experimental library"
```

Comment:
It seems sketchy to replace stock libcxx, even if it's only optional for whatever Very Good Reasons we have, with one that has experimental features enabled.